### PR TITLE
meraki - Remove YAML formatting on JSON code block

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Code block used to use YAML formatting but I removed as it's JSON in arrays and not valid. Simple code block now.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
meraki

##### ANSIBLE VERSION

```﻿
ansible 2.8.0.dev0 (meraki/scenario-code-block-fix a53366f7f0) last updated 2018/09/11 19:08:45 (GMT -500)
  config file = None
  configured module search path = ['/Users/kbreit/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/kbreit/Documents/Programming/ansible/lib/ansible
  executable location = /Users/kbreit/Documents/Programming/ansible/bin/ansible
  python version = 3.5.4 (default, Feb 25 2018, 14:56:02) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]

```

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_meraki.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ Returned Data Structures
 
 Meraki and its related Ansible modules return most information in the form of a list. For example, this is returned information by ``meraki_admin`` querying administrators. It returns a list even though there's only one.
 
-.. code-block:: yaml
+.. code-block::
```
Comment (mattclay) [inline docs/docsite/rst/scenario_guides/guide_meraki.rst]: This should probably be: `.. code-block:: json`

**Comment by kbreit** [comment]
This looks fixed. If it can be pushed, I can get this and the original PR backported to 2.7.

**Review by acozine — APPROVED** [review]

File: docs/docsite/rst/scenario_guides/guide_meraki.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/scenario_guides/guide_meraki.rst]: addressed in cf83f3d06f5067fe4b536299d871ea347b3ff8c0

**Comment by acozine** [comment]
@kbreit thanks, please ping me on the backports

**Pull request merged**